pandas/compat/__init__.py

```diff
@@ -15,6 +15,7 @@
 PY35 = sys.version_info[:2] == (3, 5)
 PY36 = sys.version_info >= (3, 6)
 PY37 = sys.version_info >= (3, 7)
+PY38 = sys.version_info >= (3, 8)
```

Comment:
Should these be equalities?

Reply:
Don't think so. We're following six's convention.